js/index.js

```diff
@@ -236,7 +237,7 @@ var Billing = Class(Emitter, function (supr) {
   this.purchase = simulatePurchase;
 });
 
-var billing = new Billing;
+var billing = new Billing();
```

Comment:
is this needed ?

Reply:
Both works the same. Not necessary though